test-monitor/nitestmonitor.yml

```diff
@@ -1150,33 +1170,43 @@ paths:
           description: Not Found
           schema:
             $ref: '#/definitions/Error'
-  /v1/query-results:
+  /v2/query-results:
     post:
       tags: [results]
-      summary: Queries test results
-      description: Uses input criteria to filter and return test results. An empty request body queries all test results.
-      operationId: query-results
+      summary: Advanced Test Result Query
+      description: Uses the Dynamic Linq query language to specify a filter and return test results. An empty request body queries all test results.
```

Comment:
Can we use a multi-line description here?   ( ">-" ) The text is wide and does not wrap.